packages/remix/src/ssr/authenticateRequest.ts

```diff
@@ -32,13 +32,15 @@ export function authenticateRequest(args: LoaderFunctionArgs, opts: RootAuthLoad
 
   const jwtKey = getEnvVariable('CLERK_JWT_KEY') || (context?.CLERK_JWT_KEY as string) || opts.jwtKey;
 
+  const apiUrl = getEnvVariable('CLERK_API_URL') || (context?.CLERK_API_URL as string);
```

Comment:
❓ Should we set a default value ` || 'https://api.clerk.dev'` or is it done already in @clerk/backend?

Reply:
It is implemented in `@clerk/backend` by you 💯